example.py

```diff
@@ -1,32 +1,33 @@
 import aflr
 
-aflr.api_key = "your-key"
-
+aflr.api_key = "vyCTlS1hLc5N5N1CON2nh2qtU6PBz7vY1MFfIlS6"
```

Comment:
please delete this api key from the api gateway. there are github bots wandering to collect api keys

Reply:
solved and deleted api-key